rllib/core/learner/learner.py

```diff
@@ -220,7 +215,7 @@ def compute_loss(self, fwd_out, batch):
                 # compute the loss based on batch and output of the forward pass
                 # to access the learner hyper-parameters use `self._hps`
 
-                return {self.TOTAL_LOSS_KEY: loss}
+                return {ALL_MODULES: loss}
```

Comment:
`compute_loss()` returns a dict mapping module IDs directly to the total loss tensors (per module AND for the ALL_MODULES key, which is a sum of all modules' loss tensors).